src/tree/AzureAccountTreeItemWithProjects.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import { Disposable, workspace, WorkspaceFolder } from 'vscode';
+import { AzExtTreeItem, AzureAccountTreeItemBase, ISubscriptionRoot, TestAzureAccount } from 'vscode-azureextensionui';
+import { tryGetFunctionProjectRoot } from '../commands/createNewProject/verifyIsProject';
+import { hostFileName } from '../constants';
+import { localize } from '../localize';
+import { getWorkspaceSetting } from '../vsCodeConfig/settings';
+import { createRefreshFileWatcher } from './localProject/createRefreshFileWatcher';
+import { LocalProjectTreeItem } from './localProject/LocalProjectTreeItem';
+import { isLocalTreeItem } from './localProject/LocalTreeItem';
+import { SubscriptionTreeItem } from './SubscriptionTreeItem';
+
+export const enableProjectTreeSetting: string = 'enableProjectTree';
+
+export class AzureAccountTreeItemWithProjects extends AzureAccountTreeItemBase {
+    private _projectDisposables: Disposable[] = [];
+
+    public constructor(testAccount?: TestAzureAccount) {
```

Comment:
Why are you passing a `TestAzureAccount` here?  Is this just so that you don't have to be logged in to use this?  I'm assuming that you're not actually expecting credentials to be in the process.env.

Reply:
This is the same `TestAzureAccount` that you implemented, just moved to a different place. It's used for the exact same reason (aka those long running tests). See [this PR](https://github.com/microsoft/vscode-azuretools/pull/499) for more info on the refactoring.